ps_facetedsearch.php

```diff
@@ -561,8 +561,8 @@ public function indexProductPrices($idProduct, $smart = true)
                     $values[] = '(' . (int) $idProduct . ',
                         ' . (int) $currency['id_currency'] . ',
                         ' . $idShop . ',
-                        ' . $minPriceValue . ',
-                        ' . $maxPriceValue . ',
+                        ' . (int) $minPriceValue . ',
+                        ' . (int) $maxPriceValue . ',
```

Comment:
Prices can have decimals, should be `(float)`?

Reply:
You're totally right :sweat_smile: